tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -60,22 +60,25 @@ When(
 )
 
 When(
-  /^"([^"]*)" (copies|moves) the following (resource|resources)$/,
+  /^"([^"]*)" (copies|moves) the following resource(?:s)? using the ([^"]*) ?(?:panel)?$/,
```

Comment:
```suggestion
  /^"([^"]*)" (copies|moves) the following resource(?:s)?( using (keyboard|drag-drop|sidebar panel))?$/,
```

Reply:
why explictly defining (keyboard|drag-drop|sidebar @saw-jan

Reply:
Because we should not accept anything as an option (they are the only ways to copy/move resources)